### PR TITLE
Fix/PC console: add a new tab that shows the Desk Rejected and Withdrawn submission

### PR DESCRIPTION
**Pull request by xkopenreview**
added a new tab to show desk rejected and withdrawn papers in pc console

### CODE REVIEW

**Comment by xkopenreview** [comment]
the new tab currently shows 3 columns: paper number; paper summary and reason (desk rejected or withdrawn)

File: openreview/conference/templates/programchairWebfield.js [hunk 1]
```diff
@@ -363,25 +365,25 @@ var getBlindedNotes = function() {
...
 };
 
-var getWithdrawnNotesCount = function() {
+var getWithdrawnNotes = function() {
   if (!WITHDRAWN_SUBMISSION_ID) {
-    return $.Deferred().resolve(0);
+    return $.Deferred().resolve([]);
   }
   return Webfield.get('/notes', {
-    invitation: WITHDRAWN_SUBMISSION_ID, limit: 1, select: 'id'
```
Comment (melisabok) [inline openreview/conference/templates/programchairWebfield.js:371]: it seems you are getting the first 1k notes, please use getAll

File: openreview/conference/templates/programchairWebfield.js [hunk 2]
```diff
@@ -877,6 +879,15 @@ var renderHeader = function() {
     }
   ];
 
+  if(ENABLE_DESK_REJECTED_WITHDRAWN_SUBMISSION_TAB) {
```
Comment (melisabok) [inline openreview/conference/templates/programchairWebfield.js:882]: I don't think this should be optional, we can always show them.

File: openreview/conference/templates/programchairWebfield.js [hunk 3]
```diff
@@ -1676,6 +1687,154 @@ var displayPaperStatusTable = function() {
 
 };
 
+var displayRejectedWithdrawnPaperStatusTable = function () {
```
Comment (melisabok) [inline openreview/conference/templates/programchairWebfield.js:1690]: Withdrawn and desk rejected submission may have reviews, reviewers or AC assigned? can we just use the same table "Paper Status"?

File: openreview/conference/templates/programchairWebfield.js [hunk 4]
```diff
@@ -1676,6 +1687,154 @@ var displayPaperStatusTable = function() {
...
+      $('.console-table th').eq(5).css('width', '12%');
+    } else {
+      $('.console-table th').eq(3).css('width', '45%');
+      $('.console-table th').eq(4).css('width', '25%');
+    }
+
+    var offset = (pageNum - 1) * PAGE_SIZE;
+    var pageData = data.slice(offset, offset + PAGE_SIZE);
+
+    if (ENABLE_REVIEWER_REASSIGNMENT) {
```
Comment (melisabok) [inline openreview/conference/templates/programchairWebfield.js:1809]: you don't need this here

File: openreview/conference/templates/programchairWebfield.js (hunk 2 above)
Comment (melisabok) [reply openreview/conference/templates/programchairWebfield.js:882]: Could add this tab at the end of the list?

**Review by melisabok — APPROVED** [review]

**Comment by xkopenreview** [comment]
@celestemartinez i added the changes you made to this pr 964f137

**Review by celestemartinez — APPROVED** [review]

**Pull request merged**